src/components/institution/AffiliationHierarchy.tsx

```diff
@@ -15,30 +15,24 @@ export const AffiliationHierarchy: FC<AffiliationHierarchyProps> = ({
   commaSeparated = false,
   boldTopLevel = true,
 }) => {
-  const [unit, isLoadingUnit] = useFetchUnitHierarchy(unitUri);
-  const unitNames = unit ? getUnitHierarchyNames(unit) : [];
+  const [department, isLoadingDepartment] = useFetchDepartment(unitUri);
+  const unitHierarchyNames = getUnitHierarchyNames(unitUri, department);
 
-  return isLoadingUnit ? (
+  return isLoadingDepartment ? (
     <AffiliationSkeleton commaSeparated={commaSeparated} />
-  ) : unit ? (
-    commaSeparated ? (
-      <i>
-        <Typography>{unitNames.join(', ')}</Typography>
-      </i>
-    ) : (
-      <div>
-        {unitNames.map((unitName, index) => (
-          <Fragment key={unitName}>
-            {index === 0 && boldTopLevel ? (
-              <Typography variant="h6">{unitName}</Typography>
-            ) : (
-              <Typography>{unitName}</Typography>
-            )}
-          </Fragment>
-        ))}
-      </div>
-    )
-  ) : null;
+  ) : commaSeparated ? (
+    <i>
```

Comment:
helt på siden : hva med "font-style: italic", siden det neppe er kursiv pga semantikken ?

Reply:
ja, sjekket faktisk det! men for en eller annen grunn var ikke det i `Typography`-komponenten, men en annen `Box`-komponent 🤔